src/test/java/co/omise/live/LiveChargeTest.java

```diff
@@ -0,0 +1,323 @@
+package co.omise.live;
+
+import co.omise.Client;
+import co.omise.ClientException;
+import co.omise.models.*;
+import co.omise.requests.Request;
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.joda.time.DateTime;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.Assert.assertTrue;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertNotNull;
+
+public class LiveChargeTest extends BaseLiveTest {
```

Comment:
Not sure should we use `LiveChargeTest` or `LiveChargeRequestTest`?

Reply:
It'll be more in line with the local tests as well, will change it.